packages/pools/src/constants/boostedPools/42161.ts

```diff
@@ -0,0 +1,8 @@
+import { BoosterConfig } from '../../utils/boosted/types'
+
+export const arbBoostedPools: BoosterConfig[] = [
+  {
+    sousId: 2,
```

Comment:
Since we already have contract address as identifier then we don't need `sousId`

Reply:
Just need to avoid same contract address in different chains.

Reply:
We already have chain based configuration so it shouldn't be an issue